_assets/js/netlify/topics/makeBodyContent.js

```diff
@@ -0,0 +1,14 @@
+const md = window.markdownit();
+md.options['html'] = true;
+md.options['linkify'] = true;
+const parser = new DOMParser();
+// This function takes our body content as input, turns the markdown into HTML,
+// then loops over the html elements
+const makeBodyContent = (bodyContent) => {
```

Comment:
nit: use function syntax?

```suggestion
function makeBodyContent(bodyContent) {
```